src/components/ProjectParticipants/ProjectParticipants.tsx

```diff
@@ -24,10 +27,21 @@ const StyledListContianer = styled.div`
 export const ProjectParticipants: FC<{
     project: NonNullable<ProjectByIdReturnType>;
 }> = ({ project }) => {
+    const [userIdToRemove, setUserIdToRemove] = useState<string>();
     const filterIds = useMemo(() => project.participants.map(({ id }) => id) ?? [], [project]);
 
     const { updateProject } = useProjectResource(project.id);
 
+    const { data: userToRemoveGoals, status } = trpc.project.getActivityGoals.useQuery(
+        {
+            id: project.id,
+            ownerId: userIdToRemove as string,
```

Comment:
Drop type casting and pass default empty string into `useState` function call

Reply:
fixed